docs/getting_started/hopsworksai/guides/getting_started_azure.rst

```diff
@@ -0,0 +1,289 @@
+Ensure that you have the Azure CLI installed (`Install the Azure CLI <https://docs.microsoft.com/en-us/cli/azure/install-azure-cli>`_)
+and are logged in (`Sign in with Azure CLI <https://docs.microsoft.com/en-us/cli/azure/authenticate-azure-cli>`_).
+
+Copy the Azure CLI command to add a service principal for Hopsworks.ai from the first step and open a terminal:
+
+.. _connect-azure-1.png: ../../../_images/connect-azure-1.png
+.. figure:: ../../../imgs/hopsworksai/connect-azure-1.png
+    :alt: Connect your Azure Account
+    :target: `connect-azure-1.png`_
+    :align: center
+    :figclass: align-center
+
+Paste the command into the terminal and execute it:
+
+.. _connect-azure-2.png: ../../../_images/connect-azure-2.png
+.. figure:: ../../../imgs/hopsworksai/connect-azure-2.png
+    :alt: Add service principal
+    :target: `connect-azure-2.png`_
+    :align: center
+    :figclass: align-center
+
+At this point, you might get the following error message.
+This means that your Azure user does not have sufficient permissions to add the service principal.
+In this case, please ask your Azure administrator to add it for you or give you the required permissions.
+
+.. code-block:: bash
+
+    $ az ad sp create --id d4abcc44-2c40-40bd-9bba-986df591c28f
+    When using this permission, the backing application of the service principal being created must in the local tenant
+
+Step 1.2: Creating a custom role for Hopsworks.ai
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Proceed to the Azure Portal and open either a *Subscription* or *Resource Group* that you want to use for Hopsworks.ai.
+Select *Add* and choose *Add custom role*. Granting access to a *Subscription* will grant access to all *Resource Groups*
+in that *Subscription*. If you are uncertain if you want that then start with a *Resource Group*.
+
+.. _connect-azure-3.png: ../../../_images/connect-azure-3.png
+.. figure:: ../../../imgs/hopsworksai/connect-azure-3.png
+    :alt: Add custom role
+    :target: `connect-azure-3.png`_
+    :align: center
+    :figclass: align-center
+
+Name the role and proceed to *Assignable scopes*:
+
+.. _connect-azure-4.png: ../../../_images/connect-azure-4.png
+.. figure:: ../../../imgs/hopsworksai/connect-azure-4.png
+    :alt: Name custom role
+    :target: `connect-azure-4.png`_
+    :align: center
+    :figclass: align-center
+
+Ensure the scope is set to the *Subscription* or *Resource Group* you want to use.
+You can change it here if required. Proceed to the JSON tab:
+
+.. _connect-azure-5.png: ../../../_images/connect-azure-5.png
+.. figure:: ../../../imgs/hopsworksai/connect-azure-5.png
+    :alt: Review assignable scope
+    :target: `connect-azure-5.png`_
+    :align: center
+    :figclass: align-center
+
+Select *Edit* and replace the *actions* part of the JSON with the one from Hopsworks.ai Azure account configuration workflow.
+Press *Save*, proceed to *Review + create* and create the role:
```

Comment:
Reading the dock, at this point, I have no idea what `Hopsworks.ai Azure account configuration workflow` is. I think I would put a new screenshot.

Reply:
done